tests/test_cache_memoize.py

```diff
@@ -202,3 +202,19 @@ def runmeonce(a):
     assert len(calls_made) == 2
     runmeonce_locmem(10)
     assert len(calls_made) == 2
+
+
+def test_cache_memoize_empty_placeholder():
+    calls_made = []
+
+    @cache_memoize(10)
+    def runmeonce(a):
+        calls_made.append(a)
+
+    result = runmeonce(20)
+    assert len(calls_made) == 1
+    assert result is None
+    result = runmeonce(20)
```

Comment:
Why this line?

Reply:
To ensure that on repeated call we get from cache the same result as on first. We actually can remove line `runmeonce(20)`. That's just to reassure that even on 3 calls we're still OK.

Reply:
If it works for 2 it should always work for 3 too. And 4. And 5.

Reply:
Agreed. Will update PR tomorrow.